<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profile_widget</class>
 <widget class="QWidget" name="profile_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Travelfimov | Профиль</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>379</width>
     <height>278</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="profile_widget_layout">
    <item row="0" column="1" rowspan="2">
     <widget class="QPushButton" name="editRouteButton">
      <property name="text">
       <string>Редактировать путь</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="backButton">
      <property name="text">
       <string>Вернуться назад</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="deleteRouteButton">
      <property name="text">
       <string>Удалить путь</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="5">
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
